rootfs/api/serializers.py

```diff
@@ -256,10 +253,11 @@ def validate_domain(self, value):
         if 'xip.io' in value:
             return value
 
+        # Let wildcards through by not trying to validate it
         if labels[0] == '*':
```

Comment:
I think we'll also need to add a check here if the length of labels is == 1 along with a unit test. If that returns true, it means the user tried `deis domains:add *`, which if I'm reading right will be accepted and the router would use it as the wildcard domain (in other words, all "failed" requests would fall back to this application). We don't want that to be an accepted value. :)